crates/ruma-events/src/image_pack.rs

```diff
@@ -0,0 +1,103 @@
+//! Types for image packs in Matrix ([MSC2545]).
+//!
+//! [MSC2545]: https://github.com/matrix-org/matrix-spec-proposals/pull/2545
+
+use std::collections::{BTreeMap, BTreeSet};
+
+use ruma_common::{serde::StringEnum, OwnedMxcUri, OwnedRoomId};
+use ruma_macros::EventContent;
+use serde::{Deserialize, Serialize};
+
+use crate::{room::ImageInfo, PrivOwnedStr};
+
+/// The content of an `im.ponies.room_emotes` event.
```

Comment:
Could we also document the stable `m.image_pack` stable identifier? And the value of the state key.

You also need to add variants in the `enums` module of the crate.